doc/STYLE_GUIDE.md

```diff
@@ -13,13 +13,9 @@
 * American English spelling is preferred. "Capitalize" vs. "Capitalise",
   "color" vs. "colour", etc.
 * Use [serial commas][].
-* Generally avoid personal pronouns in reference documentation ("I", "you",
-  "we").
-  * Pronouns are acceptable in more colloquial documentation, like guides.
```

Comment:
I would replace `Generally` with `Try to`. It should not prevent something from landing, but _it wuould be best fixed_.